### DISCUSSION TITLE
babel paser cannot paser `import type {Element} from 'react-devtools-shared/src/frontend/types';`

### DISCUSSION THREAD

**Discussion by HJX-zhanS** [babel/babel#16432] [Q&A] [answered]
When I try to use `parser.parse("import type {Element} from 'react-devtools-shared/src/frontend/types';", {sourceType: "unambiguous"});` to parse the statement, I encounter an error `Unexpected token, expected "from"`.

Is there any problem?

Someone can help me? Thanks a lot.

**Comment by liuxingbaoyu** [comment, marked as answer]
https://babeljs.io/docs/babel-parser#plugins
Please enable the corresponding `typescript` or `flow` plugin.

**Reply by HJX-zhanS** [reply]
It works, thank you!
